ui/model/asset-balance.ts

```diff
@@ -0,0 +1,7 @@
+export interface AssetBalance {
```

Comment:
Why have all those under a `model/` directory? I don't mind. Makes sense, to have all classed modeling objects or interfaces in a collected place. Just wondering how come you did that. Is it a standard thing to do?

Reply:
I am not really sure. In angular we usually have some directories to separate structure of code based on functionality, so there is a folder for components/services/models so it made sense to me to have these files in their own directory.